sdk/iot/hub/samples/src/iot_hub_twin_sample.c

```diff
@@ -286,7 +285,7 @@ static int send_get_twin()
   printf("Requesting twin document\n");
 
   if ((rc = az_iot_hub_client_twin_document_get_publish_topic(
-           &client, get_twin_topic_request_id, get_twin_topic, sizeof(get_twin_topic), NULL))
+           &client, twin_request_id++, get_twin_topic, sizeof(get_twin_topic), NULL))
```

Comment:
Any risk/concern with undetected `uint32_t` overflow?

Reply:
Highly unlikely. And at that point if it does then the message which also had that request id was 4,294,967,295 messages ago and it's very unlikely someone will be correlating that one.

Reply:
@danewalton did you already verify rid=0 works as expected?